enterprise/api_client/enterprise.py

```diff
@@ -56,6 +56,7 @@ def get_enterprise_course_runs(self, enterprise_customer):
                 self.ENTERPRISE_CUSTOMER_CATALOGS_ENDPOINT,
                 resource_id=str(enterprise_customer_catalog.uuid),
                 traverse_pagination=True,
+                querystring=500,  # we need to fetch data in large chunks so that we do not hit api limit.
```

Comment:
This should be `querystring={'page_size': 5000}`, no?

Reply:
The tricky part is going to be getting the EnterpriseCustomerCatalog detail endpoint to use the page_size parameter when pulling the content items from the discovery service [here](https://github.com/edx/edx-enterprise/blob/master/enterprise/api/v1/serializers.py#L205).

Reply:
you mean `querystring={'page_size': 500}`  right ?. 5000 may cause `Timeout`.

Reply:
>The tricky part is going to be getting the EnterpriseCustomerCatalog detail endpoint to use the page_size parameter when pulling the content items from the discovery service [here](https://github.com/edx/edx-enterprise/blob/master/enterprise/api/v1/serializers.py#L205).

I think it should work by just this update as `request.GET` would already have `page_size` parameter. I tested this locally and with a very small data set, it is working. I will deploy it the sandbox for further testing but I think more accurate testing will be performed only on stage where we have large data sets.

Reply:
never mind, I saw you comment on the ticket